frontend/packages/integration-tests-cypress/cypress.config.js

```diff
@@ -0,0 +1,32 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  viewportWidth: 1920,
+  viewportHeight: 1080,
+  screenshotsFolder: '../../gui_test_screenshots/cypress/screenshots',
+  videosFolder: '../../gui_test_screenshots/cypress/videos',
+  video: true,
+  reporter: '../../node_modules/cypress-multi-reporters',
+  reporterOptions: {
+    configFile: 'reporter-config.json',
+  },
+  fixturesFolder: 'fixtures',
+  defaultCommandTimeout: 30000,
+  retries: {
+    runMode: 1,
+    openMode: 0,
+  },
+  e2e: {
+    // TODO: see https://issues.redhat.com/browse/CONSOLE-3781
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      // eslint-disable-next-line global-require
+      return require('./plugins/index.js')(on, config);
+    },
+    specPattern: 'tests/**/*.cy.{js,jsx,ts,tsx}',
```

Comment:
FWIW, although `.cy` is the recommended file extension for cypress files with this later version, to minimize the amount of changes for the migration initially, these config files could point to `.spec` extensions instead (aka, keep them as-is), and the file names could be updated as a separate change later to meet the latest cypress standards.

Reply:
That a good point, but I'd rather do it now since that change came with Cypress 10 and have it dealt with.  The renaming is done automatically by Cypress on first launch with the 10+ version.  We'd have to manually go back and do it later.